Makefile

```diff
@@ -342,11 +333,12 @@ tar: compile-with-docker
 
 clean-tar:
 	@rm -f $(TAR_LOC)/*.$(TAR_EXT)
-	@rm -f ${VERSION_FILE}
 
 # GITHUB_USER and GITHUB_TOKEN are needed be set to run github-release
-release: tar
-	TAR_FILENAME=$(TAR_FILENAME) TAR_FILE=$(TAR_FILE) \
-	OLD_VERSION=${OLD_VERSION} BUILD_VERSION=${BUILD_VERSION} \
+release-built-version: tar
+	TAR_FILENAME=$(TAR_FILENAME) TAR_FILE=$(TAR_FILE) OLD_VERSION=${OLD_VERSION} \
 	NIGHTLY_RELEASE=${NIGHTLY_RELEASE} scripts/release.sh
 	@make clean-tar
+release: export BUILD_VERSION=$(shell cat version/CURRENT_VERSION)
```

Comment:
This seems wrong.

Reply:
this is not a target:
https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html